cli/Valet/Site.php

```diff
@@ -173,9 +172,9 @@ function proxies()
         })->reject(function ($host, $site) {
             // If proxy host is null, it may be just a normal SSL stub, or something else; either way we exclude it from the list
             return $host === '(other)';
-        })->map(function ($host, $site) use ($certs) {
+        })->map(function ($host, $site) use ($certs, $tld) {
             $secured = $certs->has($site);
-            $url = ($secured ? 'https': 'http').'://'.$site;
```

Comment:
This seems like a reasonable change, but was it really just not showing TLD in the previous incarnation? Or do we have something that shows differently depending on  the context?

Reply:
I think it really was not showing it and we just hadn't caught it. So many URLs. :) Throughout the `Site` class there is a lot of back-and-forth (or so it seems/feels) where something needs the tld vs not.